hudi-common/src/main/java/org/apache/hudi/common/fs/FSUtils.java

```diff
@@ -507,6 +507,26 @@ public static FileStatus[] getAllDataFilesInPartition(FileSystem fs, Path partit
     }
   }
 
+  /**
+   * Get the path which has a specified commit and file group id.
+   */
+  public static FileStatus getBaseFile(FileSystem fs, Path partitionPath, String fileGroupId, String commitTime) {
+    PathFilter pathFilter = path -> path.getName().startsWith(fileGroupId)
+        && path.getName().endsWith(commitTime + HoodieFileFormat.PARQUET.getFileExtension());
+    try {
+      FileStatus[] statuses = fs.listStatus(partitionPath, pathFilter);
```

Comment:
this fs listing and string matching is a performance hit. can you look into other APIs like fs view to find a performant way to do this?

Reply:
ok.